BrainPortal/public/javascripts/cbrain.js

```diff
@@ -790,6 +790,56 @@
       tool_tip.hide();
     });
 
+
+    // Adds copy to buffer button to all the element of class 'copy-text'.
+
+    if (navigator.clipboard) {
+      $(".copy-text").each(function () {
+        let $this = $(this);
+
+        // Keep line breaks
+        let text = $this.html()
+            .replace(/<br\s*\/?>/g, "\n")
+            .replace(/<\/p>\s*<p>/g, "\n")
+            .replace(/<\/div>\s*<div>/g, "\n")
+            .replace(/<\/?[^>]+(>|$)/g, "")
+            .trim();
+
+        if ($this.find(".copy-button").length === 0) {
+          let copyBtn = $(`
+                    <button class="copy-button">                    
+                        copy 
+                        <svg width="12" height="12" viewBox="0 0 24 24" fill="none" 
+                             xmlns="http://www.w3.org/2000/svg" style="vertical-align: middle;">
+                            <rect x="4" y="4" width="14" height="14" rx="2" ry="2" 
+                                  fill="#d1d5da" stroke="#6a737d" stroke-width="1.25"/>
+                            <rect x="7" y="7" width="14" height="14" rx="2" ry="2" 
+                                  fill="#d1d5da" stroke="#6a737d" stroke-width="1.25"/>
+                        </svg>
+                    </button>
+                `);
+
+          // Attach tooltip for feedback
+          let tooltip = $('<span class="copy-tooltip">Copied!</span>');
+          $this.css("position", "relative");
+          $this.append(tooltip);
+
+          // Attach the button
+          $this.append(copyBtn);
+
+          copyBtn.on("click", function (event) {
+            event.stopPropagation(); // Allow normal text selection
+            navigator.clipboard.writeText(text).then(() => {
+              console.log("Copied");
```

Comment:
Don't console.log() in a PR. Use it if you want to debug, but make you not to leave these around when you submit a PR